metricsbp/baseplate_hooks.go

```diff
@@ -35,7 +35,7 @@ type SpanHook struct {
 }
 
 func newSpanHook(prefix string, metrics Statsd, span *tracing.Span) SpanHook {
-	name := fmt.Sprintf("%s.%s.%s", prefix, span.Type(), span.Name)
+	name := fmt.Sprintf("%s.%s.%s", prefix, span.Type().String(), span.Name)
```

Comment:
nit: If the format `%v` will auto call the value's `.String()` (if available) so you can do `Sprintf("%s.%v.%s", prefix, span.Type(), span.Name)` instead.